### DISCUSSION TITLE
Upgrade path - Can/Shall I upgrade directly from v1.17.2 to v1.18.1

### DISCUSSION THREAD

**Discussion by Lineflyer** [osTicket/osTicket#6679] [Q&A] [answered]
I am currently running OSTicket v1.17.2 and would like to upgrade to the newest release.
Seeing that v1.18.1 now also support PHP 8.1 + 8.2 I would prefer to go directly to v.1.18.x

Questions:
- Why are there two different versions v1.17.5 and v1.18.1 listed as current in parallel?
- Is there any functional difference between those?
- Is there any limitation of upgrade paths?

**Comment by JediKev** [comment, marked as answer]
@Lineflyer 

Please use [the Forum](https://forum.osticket.com/) for any questions like this in the future.

osTicket always allows you to upgrade from any version that is v1.6 (released circa 2009) and above to the latest releases. You should always take full backups of the code and database before running the upgrade. Then you just simply [follow the upgrade docs](https://docs.osticket.com/en/latest/Getting%20Started/Upgrade%20and%20Migration.html). It's important to note that you need to make sure when you replace the existing code with the new release you also replace any existing plugins with their new builds before running the upgrader.

We support two major versions at a time to give people time to upgrade to the latest. Call it a courtesy if you will.

There is not much of a functional difference between 1.17.x and 1.18.x. You can always look at the [Release Notes](https://github.com/osTicket/osTicket/releases) to see the major differences between versions.

There are no limitations unless you are using a version before v1.6 (released circa 2009).

Cheers.

**Reply by Lineflyer** [reply]
Thanks for your answer and link to the forum. I totally forgot about this. Sorry!
